src/Model/Email/HoldEmail.php

```diff
@@ -26,9 +27,12 @@ public function getFromAddress()
      */
     public function getToAddress()
```

Comment:
Good point. On the other hand, we'll probably also have to rename `getFromAddress()` to `getFromEmailAddress()`. The renaming would also need to be done in the parent `Email` class since these are inherited methods.